README.md

```
@@ -120,56 +233,52 @@ $ nix develop
+# Clippy
 nix develop -c cargo clippy --workspace -- -D warnings
 
-# Run all checks
+# Everything
 nix develop -c cargo fmt --check && nix develop -c cargo clippy --workspace -- -D warnings
 ```
 
-### Workspace Structure
+### Workspace Layout
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)